adal/authentication_context.py

```diff
@@ -36,6 +36,9 @@
 from . import log
 from .constants import OAuth2DeviceCodeResponseParameters
 
+
+warnings.simplefilter('default', DeprecationWarning)  # Makes sure ADAL warnings are shown
+

```

Comment:
TBD: This line makes the warning visible to end users too. We may choose to remove this line, so that only the developers who are supposed to test their work by "python -W all ..." would see it.